File: leetcode/Array/Two Sum(1)/Solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] result = new int[2];

        Loop1: for(int i=0; i< nums.length-1; i++) {
            int sum = nums[i];
            for(int j=i+1; j< nums.length; j++) {
                sum+=nums[j];
                if(sum==target) {
                    result[0] = i;
                    result[1] = j;
                    break Loop1;
                }
                sum-=nums[j];
            }
        }
        return result;
    }
}

File: leetcode/Array/Two Sum(1)/Solution.kt
class Solution {
    fun twoSum(nums: IntArray, target: Int): IntArray {
        for (i in 0..<nums.lastIndex) {
            var sum = nums[i]
            for (j in i+1..nums.lastIndex) {
                sum += nums[j]
                if (sum == target) {
                    return intArrayOf(i, j)
                }
                sum -= nums[j]
            }
        }
        return IntArray(2) { -1 }
    }
}
